### ISSUE TITLE
Error: unexpected value for SmartSens == 0x1145

### ISSUE THREAD

**Issue by phpcoder** [OpenIPC/ipc_chip_info#14]
Executing ipc_chip_info produces an error
"Error: unexpected value for SmartSens == 0x1145"
for the --sensor_id part.
It is run in a camera doorbell device 

```
# cat /proc/cpuinfo 
Processor       : ARM926EJ-S rev 5 (v5l)
BogoMIPS        : 269.10
Features        : swp half fastmult edsp java 
CPU implementer : 0x41
CPU architecture: 5TEJ
CPU variant     : 0x0
CPU part        : 0x926
CPU revision    : 5

Hardware        : hi3518ev200
```

Full ipc_chip_info output:

```
# ./ipc_chip_info 
---
chip:
  vendor: HiSilicon
  model: 3518EV200
ethernet:
  mac: "*****"
rom:
  - type: nor
    size: 8M
    block: 64K
    partitions:
      - name: boot
        size: 0x60000
      - name: kernel
        size: 0x200000
      - name: rootfs
        size: 0x280000
      - name: rom
        size: 0x50000
      - name: app
        size: 0x2d0000
Error: unexpected value for SmartSens == 0x1145
```

Does it mean, that this type of sensor is not supported by OpenIPC?

**Comment by widgetii** [comment]
It means that we hadn't such sensor in our lab. Do you know what's a model of board? Maybe a description or even a datasheet?

**Comment by phpcoder** [comment]
It is not actually a camera, it is a popular video doorbell, but compatible with for example Yoosee firmware. It supports ONVIF streams, etc. It may add the whole class of new devices for your project. Doorbells users may have even more reasons to detach their devices from the cloud, so your project can be very useful.
All what I can find, is
- device type KW01 (board KW01-200-V1.3 2017-07-07)
- CMOS sensor 1/4 OV9712
- some spec can be seen in [https://www.moderneenergi.dk/upload/101186/doc/23702-WiFi-doorbell-KW01-specifications.pdf](url)

Let me know if you require more info. I have a root access to the system and have already got some success in modification of the original firmware.

**Comment by widgetii** [comment]
Is it still on market? Can I buy one somewhere?
To debug code I need the same equipment in my lab

**Comment by phpcoder** [comment]
Yes, it is available in some places, eg. aliexpress, but many look the same, however the one I am talking about is known as KONX KW01 gen2. As you may guess it is available in multiple chinese clones. The cheapest price I was able to find is from yoosee shop: [https://shop.yooseecamera.com/doorbellcam/yoosee-wireless-smart-intercom-doorbell-camera-poe](url)
Models can be identified with the configuration of back side connectors. The one linked here is identical to KONX KW01 gen2.
Be aware that the PoE feature of those cameras is not compatible with 802.3af 48V. It needs an included power supply with a PoE injector.
